### PR TITLE
mdconvert support netcdf

### PR DESCRIPTION
**Pull request by chrisbarnettster**
Allow conversion to netcdf format (default AMBER format).

### CODE REVIEW

**Comment by chrisbarnettster** [comment]
@tsenapathi

File: tools/mdfileconverter/md_converter.xml [hunk 1]
```diff
@@ -14,11 +14,12 @@
 
     ]]></command>
     <inputs>
-        <param argument="input_file" type="data" format='xtc, trr, dcd' label="Input file for conversion"/>
+        <param argument="input_file" type="data" format='xtc, trr, dcd, netcdf' label="Input file for conversion"/>
```
Comment (bgruening) [inline tools/mdfileconverter/md_converter.xml]: the spaces between the types should be removed.

File: tools/mdfileconverter/md_converter.xml [hunk 2]
```diff
@@ -45,11 +47,17 @@
             <param name="input_file" value="traj.dcd" ftype="dcd"/>
             <output name="output" file="traj.trr"/>
         </test>
+        <test>
+            <param name="conversion" value="traj" />
+            <param name="output_format" value="netcdf" />
+            <param name="input_file" value="traj.trr" ftype="trr"/>
+            <output name="output" file="traj.netcdf" compare="sim_size"/>
```
Comment (simonbray) [inline tools/mdfileconverter/md_converter.xml]: can you use an assertion with `has_size` here? https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-tests-test-output-assert-contents

this saves needing a large test file.

Comment (chrisbarnettster) [reply tools/mdfileconverter/md_converter.xml]: Trying it out. Then should I remove the example output file?

Comment (chrisbarnettster) [reply tools/mdfileconverter/md_converter.xml]: @simonbray Does this work with binary outputs. Tried quite a liberal delta and still get this error
```
JobOutputsError: History item  different than expected, difference (using diff):
( /tmp/tmp_jRy4Gtraj.netcdf v. /tmp/tmpxtdTLXtraj.netcdf )
Binary data detected, not displaying diff
```

Comment (bgruening) [reply tools/mdfileconverter/md_converter.xml]: @chrisbarnettster make sure you are using latest Galaxy ... or even the new-beta 20.01

Comment (chrisbarnettster) [reply tools/mdfileconverter/md_converter.xml]: Thanks I used the latest branch and looks good!  `planemo test --galaxy_branch release_20.01`

**Pull request merged**